_layouts/packages.html

```diff
@@ -0,0 +1,92 @@
+
+{% if page.authors.size > 1 %}
+    <h4>Authors</h4>
+{% else %}
+    <h4>Author</h4>
+{% endif %}
+{% assign persons = "" %}
+{% for person in page.authors %}
+    {% if person.url %}
+        {% capture persons %}{{ persons }}<a href="{{ person.url }}">{{ person.name }}</a>|{% endcapture %}
+    {% else %}
+        {% capture persons %}{{ persons }}{{person.name}}|{% endcapture %}
+    {% endif %}
+{% endfor %}
+<p>{{ persons | split: "|" | join: ", " }}</p>
+
+{% if page.maintainers.size > 1 %}
+    <h4>Maintainer</h4>
+{% else %}
+    <h4>Maintainers</h4>
+{% endif %}
+{% assign persons = "" %}
+{% for person in page.maintainers %}
+    {% if person.url %}
+        {% capture persons %}{{ persons }}<a href="{{ person.url }}">{{ person.name }}</a>|{% endcapture %}
+    {% else %}
+        {% capture persons %}{{ persons }}{{person.name}}|{% endcapture %}
+    {% endif %}
+{% endfor %}
+<p>{{ persons | split: "|" | join: ", " }}</p>
+
+<h4>Short Description</h4>
+<p>{{ page.abstract }}</p>
+
+<h4>Version</h4>
+<p> Current version shipped with GAP: {{ page.version }} &nbsp;&nbsp;(Released  {{ page.date | date: "%d/%m/%Y"}})</p>
+
+<!-- TODO href to license empty-->
+<h4>License</h4>
+<p><a href="{{ page.licenseURL }}">{{ page.license }}</a></p>
+
+<h4>Status</h4>
+<p>{{ page.status }}</p>
+
+<h4>Dependencies</h4>
+<p> GAP version: {{ page.GAP }}</p>
+<p> Needed other packages:
+{% assign pkgs = "" %}
+{% for pkg in page.needed-pkgs %}
+    {% capture pkgs %}{{ pkgs | append: pkg.name | append: "(" | append: pkg.version | append : ")" | append: "|" }}{% endcapture %}
+{% endfor %}
+{{ pkgs | split: "|" | join: ", " }}
+</p>
+<p> Suggested other packages:
+{% assign pkgs = "" %}
+{% for pkg in page.suggested-pkgs %}
+    {% capture pkgs %}{{ pkgs | append: pkg.name | append: "(" | append: pkg.version | append : ")" | append: "|" }}{% endcapture %}
+{% endfor %}
+{{ pkgs | split: "|" | join: ", " }}
+</p>
+
+<!-- TODO the link is not always correct -->
+<h4>Online documentation</h4>
+<p>{{ page.name }}:  [<a href="{{ manual }}/{{ page.doc-html }}"> HTML</a>] version&nbsp;&nbsp; [<a href="{{ manual }}/{{ page.doc-pdf }}"> PDF</a>] version&nbsp;&nbsp;</p>
+
+<!-- TODO the link to README is not always correct
+     TODO where are the file downloads? -->
+<h4>Download</h4>
+<p>
+    [<a href="{{ manual }}/README.{{ page.name | downcase }}">README</a>]&nbsp;&nbsp;&nbsp;&nbsp;{{ page.dirname }}
+    [<a href='{{ site.data.gap.gap4dist }}tar.gz/packages/{{ page.dirname }}.tar.gz'>.tar.gz&nbsp; </a>]&nbsp;&nbsp;
+    [<a href="{{ site.data.gap.gap4dist }}tar.bz2/packages/{{ page.dirname }}.tar.bz2">.tar.bz2&nbsp; </a>]&nbsp;&nbsp;
+    [<a href="{{ site.data.gap.gap4dist }}win.zip/packages/{{ page.dirname }}-win.zip">-win.zip&nbsp; </a>]&nbsp;&nbsp;
+    [<a href="{{ site.data.gap.gap4dist }}zip/packages/{{ page.dirname }}.zip">.zip&nbsp; </a>]&nbsp;&nbsp;
+</p>
+
+<!-- TODO we need a link to the github repository and the issue tracker-->
```

Comment:
Just extend your script which parses the `PackageInfo.g` to extract `IssueTrackerURL` and `SourceRepository.URL` (note that both are optional, so you need to deal with packages that don't have them gracefully)

Reply:
we will do that. This is currently WIP.

Reply:
Noted.